docs/spec.md

```diff
@@ -2084,8 +2097,20 @@ If `batch_group_count > 1`:
     * `padded_input_shape[lhs_dim] = padding[spatial_dim, 0] + dilated_input_shape[lhs_dim] + padding[spatial_dim, 1]`.
     * `dilated_window_shape[lhs_dim] = dim(rhs, rhs_dim) == 0 ? 0 : (dim(rhs, rhs_dim) - 1) * rhs_dilation[spatial_dim] + 1`.
     * `num_windows = (padded_input_shape[lhs_dim] == 0 || dilated_window_shape[lhs_dim] > padded_input_shape[lhs_dim]) ? 0 : floor((padded_input_shape[lhs_dim] - dilated_window_shape[lhs_dim]) / window_strides[spatial_dim]) + 1`.
-* (C27) element_type(`result`) $=$ element_type(`lhs`).
-* (C28) rank(`result`) $= N$.
+* (C26) rank(`result`) $= N$.
+* If the operation uses non-quantized tensors:
+  * (C27 `is_non_quantized_tensor(lhs) and is_non_quantized_tensor(rhs) and
+    is_non_quantized_tensor(result)`.
+  * (C28) element_type(`lhs`) $=$ element_type(`rhs`) $=$ element_type(`result`).
+* If the operation uses quantized tensors:
+  * (C29) `is_quantized_tensor(lhs) and is_quantized_tensor(rhs) and
+    is_quantized_tensor(result)`.
+  * (C30) `storage_type(lhs) =  storage_type(rhs)`.
+  * (C31) `expressed_type(lhs) = expressed_type(rhs) = expressed_type(result)`.
+  * (C32) `quantization_dimension(lhs)` is empty.
```

Comment:
Just for clarification, does empty quantization_dimension(tensor) mean per-tensor only quantization?

Reply:
That is right!